### ISSUE TITLE
How to add filter for PhoneNumberField

### ISSUE THREAD

**Issue by mekicha** [carltongibson/django-filter#904]
Please, is there a way I can add support to filter by phone_number. I get this error when I try to use django-filter for this:

AssertionError: CustomerFilter resolved field 'phone' with 'exact' lookup to an unrecognized field type PhoneNumberField. Try adding an override to 'Meta.filter_overrides'. See: https://django-filter.readthedocs.io/en/develop/ref/filterset.html#customise-filter-generation-with-filter-overrides

Also, the link to customize filter generation with filter overrides is broken.

code:
```
from django_filters import rest_framework as filters
from .models import Customer

class CustomerFilter(filters.FilterSet):
    name = filters.CharFilter(lookup_expr='iexact')

    class Meta:
        model = Customer 
        fields = ['name', 'phone_number']
```

`phone_number` is an instance of [django-phonenumber-field](https://github.com/stefanfoulis/django-phonenumber-field)

**Comment by carltongibson** [comment]
Fixed link: http://django-filter.readthedocs.io/en/master/ref/filterset.html

**Comment by carltongibson** [comment]
The link was fixed in 9910ee11af8ab54bd788bcf97a20f3e5c02b92b4. You should update.
